data/json/items/comestibles/carnivore.json

```diff
@@ -230,7 +231,8 @@
     "type": "COMESTIBLE",
     "copy-from": "mutant_meat",
     "name": "cooked mutant meat",
-    "description": "This is a cooked chunk of meat from a mutated animal.  It has an unsettling, spongy texture, but otherwise tastes… mostly normal.  Hopefully you got all the bits of hair and bone out…",
+    "snippet_category": "cooked_mutant_meat_desc",
+    "description": "This is a cooked chunk of meat from a mutated animal.",
```

Comment:
This description only restates the item name.  The old description _described_ the object.  Think about how the meat would look, feel, or smell different.

Reply:
The `cooked_mutant_meat_desc` strings are substituted into the description.